### PR TITLE
Change number of days for which the red risk card is shown after an exposure: UI (EXPOSUREAPP-12121)

### PR DESCRIPTION
**Pull request by jurajkusnier**
The configured number of days is changed dynamically in the following texts (see Figma)

- "Period Logged" on the red risk card
- "Period Logged" on the green risk card
- "Period Logged" on the exposure logging screen
- "Risk of Infection" on the app overview screen
- On the initial onboarding screen 
- The information popup "Information zur Funktionsweise der Risiko-Ermittlung"

### Jira Ticket
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12065

### CODE REVIEW

**Review by KathrinLuecke — DISMISSED** [review]
UA approved

File: Corona-Warn-App/src/main/res/values-bg/strings.xml [hunk 1]
```diff
@@ -201,7 +201,7 @@
     <!-- XHED: App overview subtitle for risk explanation -->
     <string name="main_overview_subtitle_risk">"Риск от заразяване"</string>
     <!-- YTXT: App overview body text about risk levels -->
-    <string name="main_overview_body_risk">"Когато регистрирането на излагания е активно, приложението изчислява какъв е Вашият риск от заразяване, като измерва продължителността и близостта на контактите Ви с хора, които са били диагностицирани с COVID-19 през последните 14 дни."</string>
+    <string name="main_overview_body_risk">"Когато регистрирането на излагания е активно, приложението изчислява какъв е Вашият риск от заразяване, като измерва продължителността и близостта на контактите Ви с хора, които са били диагностицирани с COVID-19 през последните %d дни."</string>
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/values-bg/strings.xml:204]: will this cause an issue for the translation tool?

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/onboarding/OnboardingFragmentTest.kt [hunk 2]
```diff
@@ -18,6 +26,25 @@ import testhelpers.takeScreenshot
...
 
+    @MockK lateinit var appConfigProvider: AppConfigProvider
+
+    @Before
+    fun setup() {
+        MockKAnnotations.init(this)
+
+        every { appConfigProvider.currentConfig } returns flowOf(
+            mockk<ConfigData>().apply {
+                every { maxEncounterAgeInDays } returns 14
```
Comment (LukasLechnerDev) [inline Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/onboarding/OnboardingFragmentTest.kt:37]: The new value is '10', right? Let's use this one to have the current value in the screenshots?

Comment (jurajkusnier) [reply Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/onboarding/OnboardingFragmentTest.kt:37]: I think we should use design documents from Figma as the source of truth for screenshots. The value there is 14, but I double-check with designers.

**Review by LukasLechnerDev — APPROVED** [review]
LGTM

File: Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/onboarding/OnboardingFragmentTest.kt (hunk 2 above)
Comment (jurajkusnier) [reply Corona-Warn-App/src/androidTest/java/de/rki/coronawarnapp/ui/onboarding/OnboardingFragmentTest.kt:37]: will update it in other PR if it's necessary

**Review by KathrinLuecke — APPROVED** [review]
UA approved

**Pull request merged**